config-provisioning/src/main/java/com/yahoo/config/provision/zone/ZoneApi.java

```diff
@@ -17,4 +17,6 @@
     default RegionName getRegionName() { return getId().region(); }
 
     CloudName getCloudName();
+
+    String getCloudNativeRegionName();
```

Comment:
Document what this returns for the supported clouds, AWS for now. It's the AWS availability zone you want with this right?

Reply:
No, the AWS region name, e.g. `us-east-1`